NETDefault/UnitTestingWithNUnit&Moq/ServiceTests.cs

```diff
@@ -0,0 +1,74 @@
+using Moq;
+using NUnit.Framework;
+
+namespace ServiceTests
+{
+    internal class ServiceTests : ServiceTestsBase
+    {
+        [Test]
+        public void GetDashboard_WhenCustomerNameNotFound_ItShouldThrow()
+        {
+            // Arrange
+            SetupGetUserIdByUserName(unknownUsername);
+
+            //Assert
+            Assert.Throws<NotFoundException>(() => _subject.GetDashboardByUserName(unknownUsername));
+
+            _gateway.Verify(x => x.GetUserIdByUserName(unknownUsername), Times.Once());
+            _gateway.VerifyNoOtherCalls();
+        }
+
+        [Test]
+        public void GetDashboard_WhenCustomerNameFound_ItShouldBeReturned()
+        {
+            // Arrange
+            SetupGetUserIdByUserName(knownUsername);
+
+            var expected = userId.ToString();
+
+            //Act
+            var result = _subject.GetDashboardByUserName(knownUsername);
+
+            //Assert
+            Assert.AreEqual(expected, result);
+
+            _gateway.Verify(x => x.GetUserIdByUserName(knownUsername), Times.Once());
+            _gateway.Verify(x => x.GetHobbiesByUserId(userId), Times.Once());
+        }
+
+        [Test]
+        public void GetDashboard_WhenUserFound_ItShouldGetHobbies()
+        {
+            // Arrange
+            SetupGetUserIdByUserName(knownUsername);
+            SetupGetHobbiesByUserId(userId);
+
+            //Act
+            var result = _subject.GetDashboardByUserName(knownUsername);
```

Comment:
Fixed it to a discard. And to @ThomasBleijendaal : 

_A discard communicates intent to the compiler and others that read your code: You intended to ignore the result of an expression. You may want to ignore the result of an expression, one or more members of a tuple expression, an out parameter to a method, or the target of a pattern matching expression.

Because there's only a single discard variable, that variable may not even be allocated storage. Discards can reduce memory allocations. Discards make the intent of your code clear. They enhance its readability and maintainability._

Source: https://docs.microsoft.com/en-US/dotnet/csharp/fundamentals/functional/discards